recipes/busco/meta.yaml

```diff
@@ -29,10 +30,10 @@ test:
   requires:
     - wget
   commands:
-    - run_BUSCO.py -h
-    - run_busco -h
-    - generate_plot.py -h
-    - generate_plot -h
+    - run_BUSCO.py -h >/dev/null
```

Comment:
Why is this needed @pvanheus?

Reply:
It makes for less noisy test output - not needed, but it makes spotting the source of errors easier.

Reply:
But if the error message is "missing foobar" that is printed to stdout you are not seeing it, right?
Your call. I find more output good in tests.